scipy/linalg/basic.py

```diff
@@ -1317,12 +1325,14 @@ def pinv2(a, cond=None, rcond=None, return_rank=False, check_finite=True):
     a : (M, N) array_like
         Matrix to be pseudo-inverted.
     cond, rcond : float or None
-        Cutoff for 'small' singular values.
-        Singular values smaller than ``rcond*largest_singular_value``
-        are considered zero.
```

Comment:
This older wording actually seems more readable to me. As above, you have removed the description of how `rcond` is effectively used when it is provided, in favor of describing how `None` (the default) is handled, which I think is more clearly described in a second sentence.

Reply:
indeed.